apollo-compiler/src/test/graphql/com/example/directives/TestQuery.json

```diff
@@ -1,6 +1,7 @@
 {
   "operations": [
     {
+      "filePath": "/Users/ben_schwab/airapps/apollo-android/apollo-compiler/src/test/graphql/com/example/directives/TestQuery.graphql",
```

Comment:
It seems that the apollo-codgen records this as an absolute path now. I could go and remove all these lines, but they don't seem to have an impact on the codegen.